projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -125,7 +211,8 @@ export function selectCreditCardPayment() {
 
 export function selectAccountShippingAddress() {
   const getCheckoutDetails = interceptCheckoutB2BDetailsEndpoint(
-    'b2b-checkout-replenishment/delivery-address-step/checkout-details.json'
```

Comment:
replace json file by JS object as it is easier to manipulate as we need to replace occurrences of 'addressIdFromServer' by b2bDeliveryAddress.id value.